src/requirements/data/constants.ts

```diff
@@ -1,10 +1,10 @@
 /** Course ID for special course that satisfies no requirements. */
-export const NO_EQUIVALENT_COURSES_COURSE_ID = 10;
+export const NO_EQUIVALENT_COURSES_COURSE_ID = 10; // TODO @bshen rename to NO_FULFILLMENTS_COURSE_ID
 
-/** Course ID for special course that satisfies total academic credits requirement. */
+/** @deprecated Course ID for special course that satisfies total academic credits requirement. */
 export const CREDITS_COURSE_ID = 11;
 
-/** Course ID for special course that satisfies FWS requirement. */
+/** @deprecated Course ID for special course that satisfies FWS requirement. */
 export const FWS_COURSE_ID = 12;
```

Comment:
If this is deprecated what does this mean for this data? This is what I got after running `npm run req-graph-dump hl985@cornell.edu`
![image](https://user-images.githubusercontent.com/55263191/141930282-f9fbf603-571d-48b2-8d29-66a0ddd5c067.png)

Reply:
Whoops, this was a mistake! `FWS_COURSE_ID` shouldn't be deprecated.